src/features/auth/signup/SetBirthday.tsx

```diff
@@ -110,7 +110,7 @@ export const SetBirthday: FunctionComponent<Props> = ({ route }) => {
   }, [informationModalVisible, state.date])
 
   useFocusEffect(() => {
-    // reset this variable eeach time the screen is focused to force a navigation to the next screen
+    // reset this variable each time the screen is focused to force a navigation to the next screen
```

Comment:
après coup je me crois que `is focused to prevent automatical navigation to ...`est plus approprié que 
` is focused to force ...`